appveyor.yml

```diff
@@ -1,9 +1,7 @@
 environment:
   matrix:
-    - PYTHON: "C:\\Python35"
-      TOXENV: "py35"
-    - PYTHON: "C:\\Python27"
-      TOXENV: "py27"
+    - PYTHON: "C:\\Python37"
+      TOXENV: "py37"
```

Comment:
Not sure if py35 was intentionally dropped from the windows tests?

Reply:
I assumed that we only tested one version for 2 and 3 each on Windows. I added 3.4-3.6 to synch up with Pyramid in 41e9686e4e242246482b9917a03cc5aae3a5d870.